lib/interferon/destinations/datadog.rb

```diff
@@ -15,7 +15,9 @@ def initialize(options)
         end
       end
 
-      @dog = Dogapi::Client.new(options['api_key'], options['app_key'])
+      api_timeout = options['api_timeout'] || 15
+      @dog = Dogapi::Client.new(options['api_key'], options['app_key'], nil, nil, true, api_timeout)
```

Comment:
omg `nil, nil, true`? thanks, datadog. either way, a comment would be nice here:

``ruby
# arguments are: api_key, app_key, host, device, silent, timeout
args = [
  options['api_key'],
  options['app_key'],
  nil, # host to talk to
  nil, # device
  true, # silent?
  api_timeout, #timeout 
]
@dog = Dogapi::Client.new(args)
```